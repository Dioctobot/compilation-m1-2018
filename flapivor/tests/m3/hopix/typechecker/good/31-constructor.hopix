type Color = Red | Blue | Green

val red   : Color = Red<>
val blue  : Color = Blue<>
val green : Color = Green<>