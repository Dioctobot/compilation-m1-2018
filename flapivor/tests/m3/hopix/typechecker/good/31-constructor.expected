red : Color
blue : Color
green : Color
